### DISCUSSION TITLE
How to enable hyperlinks in ModComments module ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#16200] [How to]
Hi,

In config/Security.php, I've changed this to :

````
public static $purifierAllowedDomains = false;

````

It looks like html is supported in ModComments but not hyperlinks.

![hyperlinks_modcomment](https://user-images.githubusercontent.com/2930560/175256240-852026e4-7081-48a0-b89f-ebfc51c48fab.gif)

Is there any workaround to support hyperlinks in ModComments ?

Regards,

**Comment by TonyM15** [comment]
workflows-emails also not supporting html
